pages/linux/flash.md

```diff
@@ -0,0 +1,24 @@
+# flash
+
+> Flash cards in your terminal.
+> More information: <https://github.com/tallguyjenks/fla.sh>.
+
+- Open a menu of your available flashcard decks for selection:
+
+`flash`
+
+- Display the program version:
+
+`flash -v`
+
+- Display information about the flashcard system:
+
+`flash -i`
+
+- Display a list of available commands:
+
+`flash -h`
+
+- Change the previewer from default `bat` to `cat`:
```

Comment:
Perhaps this will read better?
```suggestion
- Specify the previewer from (defaults to `bat`):
```

Reply:
@sbrl the docs says the possible options are only `bat` and `cat`

Reply:
yeah, the currently supported previewers are only those 2 at the moment. The script needs a lot of work, but hoping for more in the future. but figured most people have `cat` but `bat` is a nice shiny toy :)

Reply:
Ah ok, let's leave it as-is then.